homeassistant/components/notion/__init__.py

```diff
@@ -54,8 +49,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             entry.data[CONF_USERNAME], entry.data[CONF_PASSWORD], session
         )
     except InvalidCredentialsError:
-        LOGGER.error("Invalid username and/or password")
-        return False
+        raise ConfigEntryAuthFailed("Invalid username and/or password")
```

Comment:
This integration does not implement reauth in its config flow?

Reply:
Oof, great point. Will back that out.